docs/identities/get-started/sign-in.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: sign-in
+title: Implementing user login
+sidebar_label: Sign in
+---
+
+# Sign in
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { getSdkUrl } from "@site/src/hooks"
+import CodeBlock from "@theme/CodeBlock"
+import { FrameworkCodeTabs } from '@site/src/components/GuidesComponents'
+import jsLogin from '!!raw-loader!./_common/code-examples/js/login.js'
+import goLogin from '!!raw-loader!./_common/code-examples/go/login_handler.go'
+import nextjsLogin from '!!raw-loader!./_common/code-examples/nextjs/login.tsx'
+```
+
+The login flow authenticates existing users in your application. This guide shows how to implement a secure login process that
```

Comment:
I think this page should be structured a bit differently because it is missing how the user "triggers" the login when they e.g. click on the button top right on the app to sign in (example).

1. Explain the process (this is somewhat covered in "After successful login") high level
2. The user is redirected to, or clicks a link (`<a href="$ORY_SDK_URL/self-service/login/browser">login</a>`)
3. After login is successful, the user is redirected to whatever the redirect page is
4. Then you can do `toSession` to check the session (or redirect to the login if the session is not available).